File: RoofstockChallenge.Managers/Models/PropertyRawModel.cs
namespace RoofstockChallenge.Managers.Models
{
	public class PropertyRawModel
	{
		public int id { get; set; }
		public Address address { get; set; }
		public Financial financial { get; set; }
		public Physical physical { get; set; }
	}

	public class Address
	{
		public string address1 { get; set; }
		public string address2 { get; set; }
		public string city { get; set; }
		public string country { get; set; }
	}

	public class Physical
	{
		public int yearBuilt { get; set; }
	}

	public class Financial
	{
		public decimal listPrice { get; set; }
		public decimal monthlyRent { get; set; }
	}
}

File: RoofstockChallenge.Web/ClientApp/src/app/models/property-raw.model.ts
class AddressModel {
  public address1: string;
  public address2: string;
  public city: string;
  public country: string;
}

class PhysicalModel {
  public yearBuilt: number;
}

class FinancialModel {
  public listPrice: number;
  public monthlyRent: number;
}

export class PropertyRawModel {
  public id: number;
  public address: AddressModel;
  public financial: FinancialModel;
  public physical: PhysicalModel;
}